ui/pages/setup_passkey.tsx

```diff
@@ -129,13 +133,97 @@ const SetupPasskey: NextPage = () => {
               deduplicateValues: existingKeyNames,
             };
           }
+          if (node.meta.label?.id === ORY_LABEL_ID_ADD_SECURITY_KEY) {
+            node.meta.label.context = {
+              ...node.meta.label.context,
+              appearance: isSequencedFromSignInFlow ? "" : "positive",
+              additional: (
+                <a
+                  className="p-button--link"
+                  href="mailto:support@canonical.com"
+                >
+                  Contact support
+                </a>
+              ),
+            };
+          }
           return node;
         }),
     },
   } as SettingsFlow;
 
+  const highlightPasskeyRequirement = () => {
```

Comment:
sorry to be pedantic, but even if you define the function as a constant, you still need `useCallback` to avoid instantiating it again at every render (which will keep the address changing)

I'll approve the PR to not block it for this, but consider using useCallback as the standard approach for these functions